inc/audiences/namespace.php

```diff
@@ -755,3 +764,49 @@ function build_audience_query( array $audience ) : array {
 
 	return $groups_query;
 }
+
+/**
+ * Output audience configuration data as JSON.
+ *
+ * @return array
+ */
+function get_audience_config() : array {
+	// Note this cache is cleared in save_audience().
+	$config = wp_cache_get( 'audiences', 'altis.analytics' );
+
+	if ( $config ) {
+		return $config;
+	}
+
+	/**
+	 * Limits the number of active audiences there can be in use at any one time
+	 * on the front end.
+	 *
+	 * @param int $limit The number of audiences to fetch for use client side.
+	 */
+	$limit = apply_filters( 'altis.analytics.audiences.limit', 20 );
+	$limit = absint( $limit );
+
+	$audiences = new WP_Query( [
+		'post_type' => POST_TYPE,
+		'posts_per_page' => $limit,
+		'post_status' => 'publish',
```

Comment:
Think this should have `no_found_rows => true`, or use `showposts`

Reply:
I was thinking about some sort of notification or indication that there's a limit on the number of active audiences, I'm not sure how best to approach that yet but probably in the UI so this change makes sense.